user-journey.adoc

```diff
@@ -1,62 +1,91 @@
-.. Set up a https://github.com[GitHub account]
-.. Configure a GitHub https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[API access token].
-. *Configure certificates for Red Hat Container Image Registry*
-... RHEL - Certificates are already installed. No action is required.
-... Fedora/CentOS/VM - it is required to have this package installed in order to install Red Hat certificate. This allows the docker daemon to access the Red Hat Container Image Registry
+Before you can install Istio and Fabric8, you should already have met the following prerequisites:
+
+* Fedora/CentOS OR Mac OSX OR Ubuntu with at least 6 Gb of RAM and 4 CPUs.
+* Red Hat Docker Registry with the appropriate configuration (see below for configuration instructions)
+* Know the Ethernet IP of your running docker daemon (see below for instructions for determining this IP)
+
+If you are using the Continuous Integration (GitHub) workflow, there are the following additional prerequisites:
+
+* Install the https://git-scm.com/downloads[Git SCM client].
+* Set up a https://github.com[GitHub account] if you do not already have one.
+* Generate a https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[GitHub personal access token].
+
+////
+To install the {launcher} tool on a {OpenShiftLocal}, you must provide the {launcher} tool with a GitHub personal access token. This enables the {launcher} tool to create booster applications and save them as Git repositories in your GitHub namespace.
+////
+
+If you are using the Build and Run Locally (zip) workflow, there are the following additional prerequisites:
+
+* http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[JDK 8]
+* http://maven.apache.org/download.cgi[Maven 3.5]
+
+
+=== Registry configuration
+To configure your Red Hat Container Registry, follow the steps for your operating system:
+
+* *Red Hat Enterprise Linux* - Certificates are already installed. No action is required.
+* *Fedora/CentOS* - You must have the following package installed in order to install a Red Hat certificate. This certificate lets Docker access the Red Hat Container Registry.
 +
 ----
 yum install -y ansible python-rhsm-certificates
 ----
-... Ubuntu/Debian with native Hypervisor, you will need to take a few extra steps:
+* *Ubuntu/Debian with native Hypervisor* - You must perform the following steps in order to access the Red Hat Container Registry:
 +
 ----
 wget https://rpmfind.net/linux/fedora/linux/updates/27/x86_64/Packages/p/python-rhsm-certificates-1.20.2-1.fc27.x86_64.rpm
 sudo alien python-rhsm-certificates-1.20.2-1.fc27.x86_64.rpm
 sudo dpkg -i python-rhsm-certificates_1.20.2-2_amd64.deb
 ----
-. *Install a docker daemon and configure it*
-.. CentOS/Fedora
-... Install Atomic docker package (if not yet done):
+
+=== Install and configure Docker daemon
```

Comment:
Lowercase "docker daemon" basically everywhere. This is very important from a legal perspective.